app/Http/Controllers/SitemapController.php

```diff
@@ -46,7 +46,7 @@ public function games(){
         return Sitemap::render();
     }
 
-    public function developer(){
+    public function developer() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->